wrappers/map.js

```diff
@@ -52,8 +54,17 @@ class Map extends EventEmitter {
       this.geojsonLayers[name].removeFrom(this.map)
       delete this.geojsonLayers[name]
     }
+    
+    // if clustering is defined then add a marker cluster layer
+    // else add a geoJson layer
+    if (typeof config.cluster !== 'undefined' && config.cluster) {
+      layer = new L.MarkerClusterGroup(config.cluster);
```

Comment:
Does this work if `config.cluster === true`? `true` is valid to pass into the constructor?